src/Features/Core/Portable/CodeCleanup/OrganizeUsingsSettings.cs

```diff
@@ -18,7 +18,7 @@ public OrganizeUsingsSet(OptionSet optionSet, string language)
         public OrganizeUsingsSet(bool isRemoveUnusedImportEnabled, bool isSortImportsEnabled)
         {
             IsRemoveUnusedImportEnabled = isRemoveUnusedImportEnabled;
-            IsSortImportsEnabled = IsSortImportsEnabled;
+            IsSortImportsEnabled = isSortImportsEnabled;
```

Comment:
are we missing adequate testing?  it seems like we'd want something htat is validating all this, and that each diagnostic is respected through the cleanup codepaths.